src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/RespondToClaimTest.java

```diff
@@ -44,6 +45,7 @@ public void shouldBeAbleToSuccessfullySubmitAlreadyPaidDefence() {
     }
 
     @Test
+    @Ignore("Full Admission is not implemented, please enable once done")
```

Comment:
This test passes for CCD disabled, and @geek-soft advised to have it and accepted the risk of functional tests failure for temporarily on env where we have CCD enabled.